website/src/css/customTheme.scss

```diff
@@ -22,7 +22,7 @@
   --ifm-code-font-size: 90%;
   --ifm-code-border-radius: 3px;
   --ifm-blockquote-color: var(--ifm-font-color-base);
-  --ifm-blockquote-font-size: 16px;
+  --ifm-blockquote-font-size: 15px;
```

Comment:
This variable is potentially misnamed, as it is only used by `.admonition`. Since the default font size for page content is `17px`, I'm not sure there's much justification to reduce this value.

**Optional**: I would be in favour of removing this line _and_ removing our `font-size` and `margin-bottom` overrides to the `.admonition` selector — falling back to the Docusaurus theme defaults (which are only 1px different).

Reply:
We can remove the font size overwrite, but the margin needs to stay, default `1em` which came from the internal component styles is a bit too little spacing.